lib/al/Library/String/StringUtil.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "prim/seadSafeString.h"
```

Comment:
```suggestion
#include <prim/seadSafeString.h>
```
Also, newline after includes and before code